app/main.py

```diff
@@ -1 +1,27 @@
-# write your code here
+import os
+
+
+def move_file(command: str) -> None:
+    if len(command.split(" ")) != 3 or command.split(" ")[0] != "mv":
+        return
+    file_orig, file_goal = command.split(" ")[1:]
```

Comment:
Splitting 3 times is so bad....... Also check for the same files is not exist
```suggestion
    command_args = command.split()
    if len(command_args) != 3:
        return
    
    cmd, file_orig, file_goal = command_args
```